tests/integration/test_wasm.py

```diff
@@ -0,0 +1,90 @@
+import pytest
+
+
+WASM_DATA = {
+    "stacktraces": [
+        {
+            "registers": {},
+            "frames": [
+                {
+                    "instruction_addr": "0x8c",
+                    "in_module": 0,
+                }
+            ],
+        },
+    ],
+    "modules": [
+        {
+            "type": "wasm",
+            "debug_id": "bda18fd8-5d4a-4eb8-9302-2d6bfad846b1",
+            "code_id": "bda18fd85d4a4eb893022d6bfad846b1",
+            "debug_file": "file://foo.invalid/demo.wasm",
+        }
+    ],
+}
+
+SUCCESS_WASM = {
+    "modules": [
+        {
+            "arch": "wasm",
+            "code_id": "bda18fd85d4a4eb893022d6bfad846b1",
+            "debug_file": "file://foo.invalid/demo.wasm",
+            "debug_id": "bda18fd8-5d4a-4eb8-9302-2d6bfad846b1",
+            "debug_status": "found",
+            "features": {
+                "has_debug_info": True,
+                "has_sources": False,
+                "has_symbols": True,
+                "has_unwind_info": False
+            },
+            "image_addr": "0x0",
+            "type": "wasm"
+        }
+    ],
+    "stacktraces": [
+        {
+            "frames": [
+                {
+                    "abs_path": "/Users/mitsuhiko/Development/wasm-example/simple/src/lib.rs",
+                    "filename": "src/lib.rs",
+                    "function": "internal_func",
+                    "instruction_addr": "0x8c",
+                    "lang": "rust",
+                    "lineno": 4,
```

Comment:
This seems wrong. I'm not sure where it should be, but line 4 seems very wrong. This might be a bug in symbolic.

Reply:
can you paste the code that generated the wasm file?

Reply:
This is caused by a likely bug in rustc: https://github.com/rust-lang/rust/issues/79410

Workaround in https://github.com/getsentry/symbolic/pull/291